.vscode/extensions.json

```diff
@@ -0,0 +1,16 @@
+{
+	// See https://go.microsoft.com/fwlink/?LinkId=827846 to learn about workspace recommendations.
+	// Extension identifier format: ${publisher}.${name}. Example: vscode.csharp
+
+	// List of extensions which should be recommended for users of this workspace.
+	"recommendations": [
+		"streetsidesoftware.code-spell-checker",
```

Comment:
Code Spell Checker is surprisingly useful, and helps avoid mis-spelled variable names.